.github/workflows/config.yml

```diff
@@ -195,7 +195,7 @@ jobs:
       - name: Compare samples and results
         run: |          
           pip install numpy
-          pip install pandas
+          pip install pandas==1.5.3
```

Comment:
Fixing this version to <2.0 due to seeing some weird/different stuff with `df.read_csv()`.